tutorials/npm-enterprise-with-nexus.md

```diff
@@ -0,0 +1,45 @@
+# npm Enterprise with Nexus
+
+If you are using npm enterprise for publishing your private packages and wish to use Nexus repository to setup proxy for your private registry then follow the steps below.
+
+
+* Open *http://your-host:8081/* in your browser.
+* Sign in with admin credentials ( You can crate your own user and login with that credentials)
+```User Name: admin
+Password: admin123```.
+* Click on Settings Icon from the top left menu.
+* Select *Repository->Repositories* from the left panel options.
+* Click on `Create repository` button.
+
+**Configuring Nexus as a npm repository.**
+
+1. Select `npm (proxy)` from the repository list.
+    * Add unique repository name.
+    * Check the online option ( This will allow repository to accept incoming requests).
+2. Add remote storage url.
+    Enter the url of your private registry.
+    E.g *http://your_npme-registry-url/*
+3. Save the changes.
+
+Here’s a good documentation that explains better about configuring Nexus as npm repository *http://blog.sonatype.com/using-nexus-3-as-your-repository-part-2-npm-packages*
```

Comment:
Minor correction: "This blog post provides more detail about configuring Nexus as an npm repository: "